canadian-journal-of-remote-sensing.csl

```diff
@@ -0,0 +1,209 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" demote-non-dropping-particle="never" default-locale="en-US">
+  <info>
+    <title>Canadian Journal of Remote Sensing</title>
+    <title-short>CJRS</title-short>
+    <id>http://www.zotero.org/styles/canadian-journal-of-remote-sensing</id>
+    <link href="http://www.zotero.org/styles/canadian-journal-of-remote-sensing" rel="self"/>
+    <link href="http://www.zotero.org/styles/elsevier-harvard" rel="template"/>
+    <link href="http://www.zotero.org/styles/ecology-letters" rel="template"/>
```

Comment:
Please remove one of the template links and update to the one you actually used.